extensions/amp-consent/amp-consent.md

```diff
@@ -137,28 +157,128 @@ current user to the 3rd party vendor extensions.
 
 Unlike consent state, this `shareData` is not persisted in client side storage.
 
+##### Client caching
+As mentioned in the response section, the consent state returned in the response will be stored in `localStorage` as a client
+side cache. The cached value will be used to avoid an extra server roundtrip on user's next visit. Note that this cached value, i.e user's previous consent decision ("accepted" or "rejected"), will always be respected in prior to all other things. 
+A couple of implications with this behavior:
+
+- If a user can make consent decisions elsewhere, any decision changes will be one-time off due to the client cache.
+- When the user is travelling away from a consent enforced region, the user's very first visit in the new region will still take user's previous consent decision in cache. It's the `checkConsentHref` endpoint's job to decide if the same decision should still be respected for all future visits by echoing back user's consent, or consent is no longer required for the user  by responding `consentRequire: false, expireCache: true`.
+
+If the above behavior is unwanted, publishers can return `expireCache: true` all the time to completely disable client cache, with a tradeoff on increased latency.
+
+Note that any locally collected consent decisions via AMP's prompt UI are also stored in the same client cache overriding any previous consent state.
+
+#### consentRequired
+`consentRequired`: It accepts a boolean value indicating if a consent is required. `<amp-consent>` releases the lock immediately if `consentRequired: false`. It makes sense mostly with a combination of [geoOverride](#geooverride) config so that only a certain regions require consent. 
```

Comment:
Worth mention the default value of `consentRequired` will be set to `remote` if not specified

Reply:
suggestion : replace "releases the lock" with
`<amp-consent>` will unblock components with an `UNKNOWN_NOT_REQUIRED` state  if `consentRequired: false` and there's no previous consent state stored in client cache.